sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,223 @@
+  default_value: "true"
+- name: "RUN_QE_ACCEPTANCE_TESTS"
+  description: "Run the QE 'acceptance' tests after build and basic verification success."
+  default_value: "false"
+- name: "RUN_QE_FULLFUNCTIONAL_TESTS"
+  description: "Run the QE 'fullfunction' tests after build and basic verification success."
+  default_value: "false"
+- name: "TARGET_ORIGIN_BRANCH"
+  description: "The target branch for your pull request. For instance, if you are writing a feature for OSE 3.3, this should be 'enterprise-3.3'."
+  default_value: "master"
+provision:
+  os: "rhel"
+  stage: "bare"
+  provider: "aws"
+actions:
+  - type: "host_script"
+    title: "install CI user account"
+    script: |-
+      oct prepare user
+      sed -i 's/User ec2-user/User origin/g' ./.config/origin-ci-tool/inventory/.ssh_config
+  - type: "host_script"
+    title: "install base dependencies"
+    script: |-
+      oct prepare dependencies
+  - type: "host_script"
+    title: "prepare environment"
+    script: |-
+      set +o nounset
+      EXTENDED_TESTS_ARRAY=()
+      if [[ "$RUN_EXTENDED" == "true" ]]; then
+        EXTENDED_TESTS_ARRAY+=(extended)
+      fi
+
+      EXTENDED_TESTS=""
+      for test in "${EXTENDED_TESTS_ARRAY[@]}"
+      do
+        if [[ -z "$EXTENDED_TESTS" ]]; then
+          EXTENDED_TESTS=$test
+        else
+          EXTENDED_TESTS=$EXTENDED_TESTS,$test
+        fi
+      done
+
+      if [[ -z "$ORIGIN_FORK_ID" ]]; then
+        ORIGIN_FORK_ID=openshift
+      fi
+
+      if [[ -z "$S2I_FORK_ID" ]]; then
+        S2I_FORK_ID=openshift
+      fi
+
+      if [[ -n "$TRELLO_URL" ]]; then
+        TRELLO_URL=`echo "$TRELLO_URL" | tr -d ' '`
+        if CARD_REF=`trello card_ref_from_url --card-url ${TRELLO_URL} 2>/dev/null`; then
+          TAG_NAME=fork_ami_openshift3_${CARD_REF}
+        else
+          TAG_NAME=fork_ami_openshift3_${TRELLO_URL}
+        fi
+      elif [[ -n "$FORK_ID" ]]; then
+        TAG_NAME=fork_ami_openshift3_${FORK_ID}
+      elif [[ -n "$ORIGIN_FORK_ID" ]]; then
+        TAG_NAME=fork_ami_openshift3_${ORIGIN_FORK_ID}
+      else
+        TAG_NAME=fork_ami_openshift3_default
+      fi
+  - type: "host_script"
+    title: "install golang"
+    script: |-
+      if [[ "${TARGET_ORIGIN_BRANCH}" =~ enterprise-3.[1-2] || "${TARGET_ORIGIN_BRANCH}" =~ release-1.[1-2] ]]; then
+        oct prepare golang --version=1.4.2
+        export OS_BUILD_ENV_IMAGE="openshift/origin-release:golang-1.4"
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" =~ enterprise-3.[3-4] || "${TARGET_ORIGIN_BRANCH}" =~ release-1.[3-4] ]]; then
+        oct prepare golang --version=1.6
+        export OS_BUILD_ENV_IMAGE="openshift/origin-release:golang-1.6"
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" =~ enterprise-3.5 || "${TARGET_ORIGIN_BRANCH}" =~ release-1.5 ]]; then
+        oct prepare golang --version=1.7.5 -u https://cbs.centos.org/repos/paas7-openshift-origin36-candidate/x86_64/os/
```

Comment:
These will probably work (even if they weren't the latest version) because you'll only have one repo enabled.